iree/test/e2e/tosa_ops/bitwise_and.mlir

```diff
@@ -0,0 +1,7 @@
+func @tensor() attributes { iree.module.export } {
+  %0 = iree.unfoldable_constant dense<[0, 3, 13, 7]> : tensor<4xi32>
+  %1 = iree.unfoldable_constant dense<[0, 2, 7, 0]> : tensor<4xi32>
+  %result = "tosa.bitwise_and"(%0, %1) : (tensor<4xi32>, tensor<4xi32>) -> tensor<4xi32>
+  check.expect_eq_const(%result, dense<[0, 2, 5, 0]> : tensor<4xi32>) : tensor<4xi32>
```

Comment:
Would be nice to write these literals in binary to make it more clear how the bitwise operations are working. I assume that's not supported?

Reply:
I can't do binary but I can do hex that looks like binary..... :D